src/styles/ui-playground/ui-playground.scss

```diff
@@ -8,45 +44,114 @@ $background-color: #ededed;
 
 // Component styles
 .PlaygroundApp {
-  display: flex;
-  flex-direction: row;
+  display: grid;
+  grid-template-areas:
+    'navigation'
+    'content';
 
-  max-width: 600px;
-  margin-left: auto;
-  margin-right: auto;
-  margin-top: 10px;
-  min-height: 90vh;
-}
+  &__content {
+    padding: var.$layout-space;
+  }
+
+  &__sidebar {
+    grid-area: 'navigation';
+    max-height: 20em;
+    overflow: scroll;
+    background-color: var.$grey-2;
+  }
+
+  &__sidebar-home {
+    text-align: center;
+    padding: var.$layout-space;
+  }
 
-.PlaygroundApp__sidebar {
-  display: flex;
-  flex-direction: column;
-  padding: 10px;
-  background-color: $background-color;
-  border-radius: 3px;
-  margin-right: 10px;
+  &__content {
+    grid-area: 'content';
+  }
 }
 
 .PlaygroundApp__nav-link {
-  padding: 5px;
-  font-size: 12pt;
-
+  width: 100%;
+  padding: var.$layout-space;
+  font-size: var.$font-size--subheading;
+  border-left: 5px solid transparent;
   &:hover {
-    background-color: #ddd;
+    background-color: var.$grey-3;
+    border-left: 5px solid var.$color-brand;
   }
 }
 
-.PlaygroundApp__content {
-  flex-grow: 1;
+// Temporarily enforce the body font size of the client to demonstrate how
+// components would scale in-situ
+.PatternPage {
+  font-size: 13px;
+  &__content {
+    padding: 2em 0;
+  }
 }
 
-.heading {
-  font-size: $title-font;
-  border-bottom: 2px solid $background-color;
-  margin-bottom: 20px;
+.Pattern {
+  margin-bottom: 4em;
+  p {
+    margin: 1em;
+  }
+}
+
+.PatternExamples {
+  border: 1px solid var.$grey-3;
+  border-collapse: collapse;
+  width: 100%;
+  margin-bottom: 2em;
+
+  th,
+  td {
+    padding: 0.5em;
+  }
+
+  th {
+    background-color: var.$grey-1;
+    border-bottom: 1px solid var.$grey-3;
+    text-align: left;
+  }
+
+  td {
+    padding-top: 10px;
+    padding-bottom: 10px;
+  }
+
+  tr:nth-child(even) td {
+    background: var.$grey-0;
+  }
+
+  code {
+    color: var.$grey-mid;
+  }
+}
+
+.PatternExample {
+  &__example {
+    display: flex;
+    align-items: center;
+    & > * + * {
```

Comment:
Curious what this does?

Reply:
Adds a left margin to all immediate descendants except the first. Translation: there's a 1em space between immediate-child elements. This is copied from the `horizontal-rhythm` mixin.